4.0.2 tag fails to run the Bundles tests

```
[INFO] EclipseLink Bundles MOXy standalone ................ SUCCESS [  1.185 s]
[INFO] EclipseLink Bundles eclipselink.jar (non-OSGi) ..... SUCCESS [02:03 min]
[INFO] EclipseLink Bundles tests .......................... FAILURE [ 32.849 s]
[INFO] EclipseLink Test Server Parent ..................... SKIPPED
[INFO] EclipseLink JPA Test ............................... SKIPPED
```

Specifically, it fails to find lots of classes (both eclipselinbk adn external) that it should be able to find:
```
[ERROR] Failures:
[ERROR]   OSGiBundleTest.testAsmCoreVersion:75->loadClass:187 Cannot find and load class: org.eclipse.persistence.asm.AnnotationVisitor
[ERROR]   OSGiBundleTest.testAsmELVersion:69->loadClass:187 Cannot find and load class: org.eclipse.persistence.internal.libraries.asm.AnnotationVisitor
[ERROR]   OSGiBundleTest.testInternalJaxb:81->loadClass:187 Cannot find and load class: org.eclipse.persistence.internal.jaxb.AttributeNodeImpl
[ERROR]   OSGiBundleTest.testInternalJaxbMany:87->loadClass:187 Cannot find and load class: org.eclipse.persistence.internal.jaxb.many.ArrayValue
[ERROR]   OSGiBundleTest.testJavaxNaming:165->loadClass:187 Cannot find and load class: javax.naming.InitialContext
[ERROR]   OSGiBundleTest.testJavaxXmlParsers:159->loadClass:187 Cannot find and load class: javax.xml.parsers.ParserConfigurationException
[ERROR]   OSGiBundleTest.testJaxb:93->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.JAXBContext
[ERROR]   OSGiBundleTest.testJaxbAttachment:99->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.attachment.AttachmentMarshallerAdapter
[ERROR]   OSGiBundleTest.testJaxbCompiler:105->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.compiler.AnnotationsProcessor
[ERROR]   OSGiBundleTest.testJaxbDynamic:111->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.dynamic.DynamicJAXBContext
[ERROR]   OSGiBundleTest.testJaxbDynamicMetadata:117->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.dynamic.metadata.Metadata
[ERROR]   OSGiBundleTest.testJaxbJavamodel:123->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.javamodel.AnnotationProxy
[ERROR]   OSGiBundleTest.testJaxbJavamodelOxm:129->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.javamodel.oxm.OXMJavaClassImpl
[ERROR]   OSGiBundleTest.testJaxbJavamodelReflection:135->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.javamodel.reflection.AnnotationHelper
[ERROR]   OSGiBundleTest.testJaxbMetadata:141->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.metadata.MetadataSource
[ERROR]   OSGiBundleTest.testJaxbRs:147->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.rs.MOXyJsonProvider
[ERROR]   OSGiBundleTest.testJaxbXmlmodel:153->loadClass:187 Cannot find and load class: org.eclipse.persistence.jaxb.xmlmodel.JavaAttribute
[ERROR]   OSGiBundleTest.testOrgXmlSaxHelpers:171->loadClass:187 Cannot find and load class: org.xml.sax.helpers.DefaultHandler
[ERROR] Errors:
[ERROR]   BeanValidationTest.setUp:68 » ClassNotFound org.eclipse.persistence.jaxb.JAXBContextFactory not found by PAXEXAM-PROBE-6c11a3d5-e07b-43f0-9b36-1e0903215ddd [27]
[ERROR]   BeanValidationTest.setUp:68 ClassNotFound org.eclipse.persistence.jaxb.JAXBContextFactory not found by PAXEXAM-PROBE-6c11a3d5-e07b-43f0-9b36-1e0903215ddd [27]
[ERROR]   BeanValidationTest.setUp:68 ClassNotFound org.eclipse.persistence.jaxb.JAXBContextFactory not found by PAXEXAM-PROBE-6c11a3d5-e07b-43f0-9b36-1e0903215ddd [27]
[ERROR]   BeanValidationTest.setUp:68 ClassNotFound org.eclipse.persistence.jaxb.JAXBContextFactory not found by PAXEXAM-PROBE-6c11a3d5-e07b-43f0-9b36-1e0903215ddd [27]
[ERROR]   NoBeanValidationTest.setUp:57 » ClassNotFound org.eclipse.persistence.jaxb.JAXBContextFactory not found by PAXEXAM-PROBE-6c11a3d5-e07b-43f0-9b36-1e0903215ddd [22]
[ERROR]   NoBeanValidationTest.setUp:57 ClassNotFound org.eclipse.persistence.jaxb.JAXBContextFactory not found by PAXEXAM-PROBE-6c11a3d5-e07b-43f0-9b36-1e0903215ddd [22]
[ERROR]   NoBeanValidationTest.setUp:57 ClassNotFound org.eclipse.persistence.jaxb.JAXBContextFactory not found by PAXEXAM-PROBE-6c11a3d5-e07b-43f0-9b36-1e0903215ddd [22]
[ERROR]   NoBeanValidationTest.setUp:57 ClassNotFound org.eclipse.persistence.jaxb.JAXBContextFactory not found by PAXEXAM-PROBE-6c11a3d5-e07b-43f0-9b36-1e0903215ddd [22]
[INFO]
[ERROR] Tests run: 26, Failures: 18, Errors: 8, Skipped: 0
[INFO]

```


Windows11 
Java 17.0.3.1
Maven 3.6.3

```
C:\eclipselink>mvn --version
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: C:\Dev\apache-maven-3.6.3\bin\..
Java version: 17.0.3.1, vendor: Oracle Corporation, runtime: C:\Dev\jdk-17.0.3.1
Default locale: en_AU, platform encoding: Cp1252
OS name: "windows 11", version: "10.0", arch: "amd64", family: "windows"
```

Also, the build instructions at https://wiki.eclipse.org/EclipseLink/Building are woefully out of date (at least for 4.0.2+).

--- a/bundles/tests/src/test/java/org/eclipse/persistence/testing/osgi/OSGITestHelper.java
+++ b/bundles/tests/src/test/java/org/eclipse/persistence/testing/osgi/OSGITestHelper.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, 2021 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2023 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
  * terms of the Eclipse Public License v. 2.0 which is available at
@@ -15,6 +15,9 @@
 package org.eclipse.persistence.testing.osgi;
 
 import org.ops4j.pax.exam.Option;
+
+import java.net.MalformedURLException;
+import java.nio.file.Path;
 
 import static org.ops4j.pax.exam.CoreOptions.bundle;
 import static org.ops4j.pax.exam.CoreOptions.cleanCaches;
@@ -41,10 +44,10 @@ public class OSGITestHelper {
 
         return options(
                 // JAXB API
-                bundle("file:" + PLUGINS_DIR + ACTIVATION_JAR),
-                bundle("file:" + PLUGINS_DIR + JAXB_API_JAR),
+                bundle(pathToUrl(PLUGINS_DIR + ACTIVATION_JAR)),
+                bundle(pathToUrl(PLUGINS_DIR + JAXB_API_JAR)),
                 //JAXB_OSGI
-                bundle("file:" + PLUGINS_DIR + JAXB_OSGI),
+                bundle(pathToUrl(PLUGINS_DIR + JAXB_OSGI)),
 
                 systemPackage("javax.rmi"),
                 systemPackage("javax.rmi.CORBA"),
@@ -52,12 +55,12 @@ public class OSGITestHelper {
                 systemPackage("org.omg.CORBA.portable"),
                 systemPackage("org.omg.CORBA_2_3.portable"),
                 // JAX-RS API
-                bundle("file:" + PLUGINS_DIR + JAXRS_JAR),
+                bundle(pathToUrl(PLUGINS_DIR + JAXRS_JAR)),
 
                 // EclipseLink bundles
-                bundle("file:" + PLUGINS_DIR + "org.eclipse.persistence.moxy.jar"),
-                bundle("file:" + PLUGINS_DIR + "org.eclipse.persistence.core.jar"),
-                bundle("file:" + PLUGINS_DIR + ASM_JAR),
+                bundle(pathToUrl(PLUGINS_DIR + "org.eclipse.persistence.moxy.jar")),
+                bundle(pathToUrl(PLUGINS_DIR + "org.eclipse.persistence.core.jar")),
+                bundle(pathToUrl(PLUGINS_DIR + ASM_JAR)),
 
                 cleanCaches(),
                 junitBundles());
@@ -86,5 +89,13 @@ public class OSGITestHelper {
                 cleanCaches(),
                 junitBundles()
         );
+    }
+
+    private static String pathToUrl(String path) {
+        try {
+            return Path.of(path).toUri().toURL().toExternalForm();
+        } catch (MalformedURLException e) {
+            throw new RuntimeException(e);
+        }
     }
 }

